pkg/serverless/daemon/interval_test.go

```diff
@@ -87,15 +93,15 @@ func TestInvocationInterval(t *testing.T) {
 	// first scenario, validate that we're not computing the interval if we only have 2 invocations done
 	// -----
 
-	for i := 0; i < 2; i++ {
+	for i := 0; i < 19; i++ {
 		time.Sleep(100 * time.Millisecond)
 		d.lastInvocations = append(d.lastInvocations, time.Now())
 		assert.Equal(time.Duration(0), d.InvocationInterval(), "we should not compute any interval just yet since we don't have enough data")
 	}
 	time.Sleep(100 * time.Millisecond)
 	d.lastInvocations = append(d.lastInvocations, time.Now())
 
-	//	assert.Equal(d.InvocationInterval(), time.Duration(0), "we should not compute any interval just yet since we don't have enough data")
+	//  assert.Equal(d.InvocationInterval(), time.Duration(0), "we should not compute any interval just yet since we don't have enough data")
```

Comment:
What is this comment?

Reply:
I don't know. I guess it can be removed.